test/node/connection-handshake.js

```diff
@@ -126,3 +126,17 @@ test.test('must not accept handshakes on a client', (test) => {
   const connection = new jstp.Connection(transport, null, {});
   transport.emitPacket(handshake);
 });
+
+test.test(
```

Comment:
@nechaido That's because there are no parallel test now, but there may be in the future, but that's up to you, I don't have a strong opinion on it.